reports/Washington DC.md

```diff
@@ -384,3 +384,17 @@ id: dc-dc-25
 * https://www.tiktok.com/@organicgoodz/video/6846083558910512389
 * https://www.tiktok.com/@organicgoodz/video/6845323150729678085
 
+
+### Multiple officers arrest peaceful protester | July 7th
+
+The following two videos are in chronological order, the first showing the protester speaking at one officer (He is not the person speaking with the megaphone, however his words are clearly audible). This turns violent as an officer shoves the protestor into the concrete, and is quickly aided by at least one other officer. 
+In the second video four officers are shown to be directly involved in the arrest with many more barricading them with bikes to keep the crowd back.
```

Comment:
could you describe the arrest a little more clearly? From the second video it appears that the officer is laying on top of the arrestee.

Reply:
I'm not entirely sure it can be said that they are laying on him, the best view of that was at approx 6s and that is an unclear view at best, unfortunately I wasn't able to find any other angles. Then later at the 30s mark it is clear that the officers are not putting excessive weight on him. That said I agree that the description should have had a note that the man was arrested on the ground, so I added that.
(timestamps are in the 2nd video)